Bottom: 75fef7778d5397443fdec18f035348f7320a5047
Top:    6c0129f352c2cd3c1851b4dd8f2dfde46fb37074
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-05-12 11:25:12 +0800

Refresh of vhost_net-introduce-helper-to

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 18cd90e..15d191a 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -459,11 +459,11 @@ static bool vhost_exceeds_maxpend(struct vhost_net *net)
 	       min_t(unsigned int, VHOST_MAX_PEND, vq->num >> 2);
 }
 
-static int init_iov_iter(struct vhost_virtqueue *vq, struct iov_iter *iter,
-		         int hdr_size, int out)
+static size_t init_iov_iter(struct vhost_virtqueue *vq, struct iov_iter *iter,
+			    size_t hdr_size, int out)
 {
 	/* Skip header. TODO: support TSO. */
-	int len = iov_length(vq->iov, out);
+	size_t len = iov_length(vq->iov, out);
 
 	iov_iter_init(iter, WRITE, vq->iov, out, len);
 	iov_iter_advance(iter, hdr_size);
